src/features/items/queries.js

```diff
@@ -1,10 +1,13 @@
 import { useQuery } from 'react-query';
 import doFetchItems from './do-fetch-items';
+import { langCode } from '../../modules/lang-helpers';
+
+import defaultItems from '../../data/items.json';
 
 export const useItemsQuery = (queryOptions) => {
-    const itemsQuery = useQuery('items', () => doFetchItems(), {
+    const itemsQuery = useQuery('items', () => doFetchItems(langCode()), {
         refetchInterval: 600000,
-        placeholderData: [],
+        placeholderData: defaultItems,
```

Comment:
`placeholderData` can be a function that loads our static data and removes flea prices, I think this can simplify many parts of the code that you nedded to add to do the cached version of fetching

Reply:
The cached version fetching code is only run once at the prebuilt stage. To me, it makes sense to modify the data at that point so no further processing is needed and it can be provided immediately at runtime. But I may be misunderstanding your point.

Reply:
Although this does give me an idea about caching additional languages...

Reply:
You are right, I was thinking about another thing, if it's run in prebuild once then it's perfect